lighthouse-treemap/README.md

```diff
@@ -0,0 +1,12 @@
+# Lighthouse Treemap Viewer
+
+## Development
+
+```sh
+yarn serve-treemap
+
+# in separate terminal, start build watch
+find lighthouse-treemap | entr -s 'DEBUG=1 yarn build-treemap'
```

Comment:
is this a new dependency we need to add to CONTRIBUTING.md?

```bash
$ which entr
entr not found
```

Reply:
No, that doesn't seem important to add there. I'll just add a comment here `brew install entr`.